error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/CharSequenceRules.java

```diff
@@ -0,0 +1,30 @@
+package tech.picnic.errorprone.refasterrules;
+
+import com.google.errorprone.refaster.Refaster;
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.AlsoNegation;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import tech.picnic.errorprone.refaster.annotation.OnlineDocumentation;
+
+/** Refaster rules related to expressions dealing with {@link CharSequence}s. */
+@OnlineDocumentation
+final class CharSequenceRules {
+  private CharSequenceRules() {}
+
+  /**
+   * Prefer {@link CharSequence#isEmpty()} over alternatives that consult the char sequence's
+   * length.
+   */
+  static final class CharSequenceIsEmpty {
+    @BeforeTemplate
+    boolean before(CharSequence ch) {
```

Comment:
```suggestion
    boolean before(CharSequence charSequence) {	
```
As `ch` generally denotes "char"`. (Unifying the variable naming approach is also on my list, somewhere ;).)